app/Files/File.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
Noted but arguable. It depends on how complicated the logic is and how many sections are using the functions. It's a trade-off between being intentional and keeping it simplistic.